build/tools/esp32/esptool_py/romfs_flash_address.sh

```diff
@@ -31,11 +31,11 @@ i=0
 #find romfs partition number
 for pname in $flash_type_list
 do
-	if [ "$pname" == "romfs" ]; then
+	if [ "$pname" = "romfs" ]; then
 		isromfs_exist=1
 		break	
 	fi
-	let "romfs_num += 1"			
+	romfs_num=`expr $romfs_num + 1`
```

Comment:
Please provide space before and after  '='
like:	romfs_num = `expr $romfs_num + 1`

Reply:
I checked all exist shell files using "expr" in TizenRT,  no need  to add space here. 
Such as
TizenRT/build/configs/artik05x/scripts/partition_gen.sh
TizenRT/build/configs/sidk_s5jt200/tools/openocd/partition_gen.sh

could you show the shell program rules guide in TizenRT?

Reply:
yes I checked it is not there.
Keep it as it is.